bitnami/zookeeper/templates/configmap.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     chart: {{ template "zookeeper.chart" . }}
     release: {{ .Release.Name | quote }}
     heritage: {{ .Release.Service | quote }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
```

Comment:
I'd use `{{ .Release.Name | quote }}` for consistency with other labels. This comment applies to every manifest file.

Reply:
Done. I also removed the `component` label again, as it will be implemented by #1127